apps/aecore/src/aec_chain_state.erl

```diff
@@ -245,6 +270,8 @@ assert_previous_height(Node) ->
 %% To assert the target calculation we need DeltaHeight headers counted
 %% backwards from the node we want to assert. If Height <= DeltaHeight
 %% we will need all headers back to genesis.
+%% NG-INFO: micro blocks are signed, so we don't validate the target
```

Comment:
But then the signature probably should be checked? But that might have happened elsewhere.